Makefile.am

```diff
@@ -980,7 +980,7 @@ imap_libcyrus_imap_la_LIBADD = \
     lib/libcyrus.la
 
 imap_libcyrus_imap_la_CFLAGS = $(AM_CFLAGS) $(CFLAG_VISIBILITY)
-imap_libcyrus_imap_la_CXXFLAGS = $(AM_CXXFLAGS)
+imap_libcyrus_imap_la_CXXFLAGS = $(AM_CXXFLAGS) $(CFLAG_VISIBILITY)
 
```

Comment:
Does it work, when CFLAG_VISIBILITY is passed instead to imap_libcyrus_LDFLAGS and removed from …CXXFLAGS and …CFLAGS?

Reply:
Off the top of my head, probably.  But what we already have is well tested, I intend to backport this fix to the stable branches, so for now I'm keeping things as similar as possible to what we already know works.